app/views/admin/data_update_scripts/index.html.erb

```diff
@@ -19,8 +19,13 @@
           <td data-filename="<%= script.file_name %>" id="data_update_script_<%= script.id %>_filename" class="data_update_script__filename">
             <%= script.file_name %>
             <% if script.status == "failed" %>
-              <button class="ml-2" id="data_update_script_<%= script.id %>_button"
- data-action="click->data-update-script#forceRun" data-value="<%= script.id %>" type="button" className="crayons-btn crayons-btn--secondary">
+              <button type="button"
+                      class="crayons-btn crayons-btn--s crayons-btn--secondary"
+                      id="data_update_script_<%= script.id %>_button"
+                      data-action="click->data-update-script#forceRun"
+                      data-value="<%= script.id %>"
+                      data-force-run-path="<%= force_run_admin_data_update_script_path(script.id) %>"
+                      data-index-path="<%= admin_data_update_script_path(script.id) %>">
```

Comment:
**non-blocking**

I am probably missing something here, but is `data-index-path` used?

Reply:
Nooooo i thought i removed it :( I'll remove it now